packages/salesforcedx-sobjects-faux-generator/src/generator/configUtil.ts

```diff
@@ -9,23 +9,33 @@ import {
   ConfigAggregator,
   ConfigFile,
   ConfigValue,
-  StateAggregator
+  OrgConfigProperties
 } from '@salesforce/core';
 import * as path from 'path';
 
-const defaultUserNameKey = 'defaultusername';
+async function getConfigAggregator(
+  projectPath: string
+): Promise<ConfigAggregator> {
+  const origCurrentWorkingDirectory = process.cwd();
+  // Change the current working directory to the project path,
+  // so that ConfigAggregator reads the local project values
+  process.chdir(projectPath);
+  const configAggregator = await ConfigAggregator.create();
+  // Change the current working directory back to what it was
+  // before returning
+  process.chdir(origCurrentWorkingDirectory);
+  return configAggregator;
+}
 
 export class ConfigUtil {
```

Comment:
sorry dumb q.  Can we just update this package to remove this configUtil and use the one in utils? Or is that the scope creep we're hoping to avoid?

Reply:
Beginning to do that sort of work is where I started running into issues before.  However, this is a smaller scoped set of changes than last time, and the build has evolved.  I'll try it and report back.

Reply:
This package doesn't have a dep on the utils package like the core package does.  Could that be why importing from utils wouldve caused issues last time?

Reply:
https://salesforce-internal.slack.com/archives/G02JDKH7W9K/p1660866255590169